test/e2e/validation_test.go

```diff
@@ -481,9 +481,13 @@ func testServicesConfigMap(t *testing.T) {
 		require.NoError(t, err, "unable to parse ConfigMap data")
 
 		// Check that the expected services are defined within the CM data
```

Comment:
So this is doing more than just what the comment says. I suggest updating the comment.